runtime/doc/lsp.txt

```diff
@@ -1469,6 +1462,57 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (integer)
 
 
+==============================================================================
+Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
+
+get({filter})                                       *vim.lsp.inlay_hint.get()*
+    Get the list of inlay hints, (optionally) restricted by buffer, client, or
+    range.
+
+    Example usage: >lua
+        local timeout, bufnr = 100, vim.api.nvim_get_current_buf()
+        local hint = vim.lsp.inlay_hint.get({ bufnr = bufnr })[1]
+
+        local client = vim.lsp.get_client_by_id(hint.client_id)
+        resolved_hint = client.request_sync(
+          'inlayHint/resolve', hint.inlay_hint, timeout, bufnr).result
+        vim.lsp.util.apply_text_edits(resolved_hint.textEdits, bufnr, client.encoding)
+
+        location = resolved_hint.label[1].location
+        client.request("textDocument/hover", {
+          textDocument = { uri = location.uri },
+          position = location.range.start,
+        })
+<
+
+    Parameters: ~
+      • {filter}  vim.lsp.inlay_hint.get.filter ? Optional fields |kwargs|:
+                  • bufnr (integer?)
```

Comment:
What happened to the parameter descriptions?

Reply:
With the rename from `opts` to `filter` they became self-documenting (I think)

Reply:
Yeah, the names are descriptive enough but I think we should still call out the values with special meaning.